ember/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "hello-world",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Small description for hello-world goes here",
+  "repository": "",
+  "license": "MIT",
+  "author": "",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build",
```

```suggestion
    "build": "ember build -prod",
```

This should also be documented in the readme. Please try deploying it!